internal/digreflect/func_test.go

```diff
@@ -55,7 +56,6 @@ func TestInspectFunc(t *testing.T) {
 		// We don't match the exact file name because $GOPATH can be anywhere
 		// on someone's system. Instead we'll match the suffix.
 		wantFileSuffix string
-		wantStringLike string
```

Comment:
Why does this change mean the removal of the wantStringLike test scenarios?

Reply:
Oh, that's a rider. It was an unused field.

Reply:
Ah, okay!